base/util.go

```diff
@@ -444,6 +444,7 @@ func RetryLoop(ctx context.Context, description string, worker RetryWorker, slee
 
 	numAttempts := 1
 
+	startTime := time.Now()
```

Comment:
time.Now() isn't free, and we use RetryLoop in a lot of places, some that may be on performance critical paths.  I'm not sure the additional retry duration information on error is that useful generically.  If it's particularly useful for index initiatlization, maybe move the timer outside of the retry loop call?

Reply:
I think it's somewhat helpful for index initialization but it's trivially calculatable looking at the logs. The trickiest part looking at the logs is making sure that you look at the right collection for index initialization. Without the timeout on capella, I don't think this will be terribly useful, especially in the case cancellation.